code/game/machinery/atmo_control.dm

```diff
@@ -407,6 +407,7 @@
 
 /obj/machinery/computer/general_air_control/large_tank_control/linkWith(mob/user, obj/O, list/context)
 	if(context["slot"]=="input" && is_type_in_list(O,input_linkable))
+		input_tag = O:id_tag
```

Comment:
Do you want to get stabbed?

Reply:
I know... this crap is all over these files, and I really didn't want to mess with it. But if you want I can refactor this stuff to not use `:`